shared/surface_flux.F90

```diff
@@ -995,4 +1024,75 @@ subroutine ncar_ocean_fluxes_multilevel (u_del, t, ts, q, qs, zu, zt, zq, avail,
 
 end subroutine ncar_ocean_fluxes_multilevel
 
+subroutine iter_monin_obukhov_ocean (                      &
+           z_atm, u_atm, v_atm, w_atm, thv_atm, q_atm,     &
+           u_surf, v_surf, thv_surf, q_surf0,              &
+           rough_mom, rough_heat, rough_moist,             &
+           cd_m, cd_t, cd_q, u_star, b_star, avail, seawater)
+!~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~!
+!  Right now, it  is only effective when ocean_rough = 'hwrf17', but this
```

Comment:
You should probably check the roughness scheme and print a note to the screen if it's not `hwrf17`